packages/router/src/index.ts

```diff
@@ -259,21 +259,6 @@ export class Router<App extends Router = any, Req = any, Res = any> {
     return this
   }
 
-  /**
```

Comment:
why move it away from router?

Reply:
coverage showed the left branch was never reached
```typescript
this.parent ? this.parent.path() + this.mountpath : ''
```

Which is true because the Router by itself has no parent. Instead, the app has a parent and thus makes sense if moved to App.

Reply:
hm maybe we should assign the parent in the router in this case

Reply:
it won't if the Router by itself is initialized. For example
```typescript
const router = new Router()
router.parent // will always be null
router.path() // will be /
```

Reply:
what I mean is we can't mount the Router. So the parent of the Router can never have a non null value.

Reply:
alright got it, then moving it to app makes sense